vagrant/provisioning/roles/pentaho-configuration/tasks/main.yml

```diff
@@ -231,4 +230,12 @@
     delay: 5
     timeout: 300
   when: security_provider_updated is changed
-
+    
+- name: Fix loader not closing issue
+  become: yes
+  become_user: pentaho
+  replace:
+    backup: yes
+    path: "{{ root_folder }}/app/pentaho/pentaho-server/pentaho-solutions/system/reporting/reportviewer/compressed/reportviewer-app.js"
+    regexp: 'l\&\&l.className\&\&l'
+    replace: 'l&&l'
```

Comment:
What issue does this fix exactly ... is there an AFDP ticket?  And, you definitely don't have to escape the ampersands in the replacement string?

Reply:
https://arkcase.atlassian.net/browse/ACP-59
https://project.armedia.com/jira/browse/AFDP-8915

I needed to escape the ampersands in **sed** command. And It works from here...